k2/csrc/CMakeLists.txt

```diff
@@ -90,6 +90,8 @@ set(cuda_tests
   algorithms_test
   array_ops_test
   array_test
+  cuda_tests
```

Comment:
For each xxx_test, it's expected that there is a file xxx_test.cu

`cuda_tests` can be removed as it is just a variable here.

```
set(variable_name
  value1
  value2
  ...
)
```